app/helpers/validation/height.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from flask import abort
+from app.helpers.helpers import float_raise_nan
+
+
+def validate_lon_lat(lon, lat):
+    if lon is None:
+        raise abort(400, "Missing parameter 'easting'/'lon'")
```

Comment:
The `raise` in front of `abort` is not needed and should be removed same for other lines below

Reply:
has been removed, thanks